src/opengl/GLWin.cc

```diff
@@ -134,6 +134,15 @@ void GLWin::windowRefreshCallback(GLFWwindow *win) {
   w->dirty = true;
 }
 
+void GLAPIENTRY messageCallback(GLenum source, GLenum type, GLuint id,
```

Comment:
TODO: Add a debug level system using the `GLenum severity` for blocking out extraneous MPV errors.